### PR TITLE
feat: create login page

### PR DESCRIPTION
**Pull request by Promit-revar**
Designed login page with base components...
Screenshot 2023-03-12 at 11 44 43 AM

### CODE REVIEW

**Comment by Sukhacoder02** [comment]
Hey Promit,
Does the Mckinsey logo in the header really looks squished or its just in the screenshot?
Also, could you please center the Login div  both vertically and horizontally?

**Review by Sukhacoder02 — APPROVED** [review]

File: src/components/footer/index.jsx [hunk 1]
```diff
@@ -4,10 +4,10 @@ import mckinseyLogo from '../../assets/logo/mckinsey-logo.png';
 
 const index = () => {
   return (
-    <div className='footer'>
+    <div className="footer">
```
Comment (Chetan-MckinseyBEBR) [inline src/components/footer/index.jsx:7]: Check the eslint config, it should be single quotes

File: src/pages/LoginPage/LoginPage.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.login-page{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    background: #D9D9D9;
+}
+.card{
```
Comment (Chetan-MckinseyBEBR) [inline src/pages/LoginPage/LoginPage.css:9]: have a new empty line separating the code sections

File: src/pages/LoginPage/index.jsx [hunk 3]
```diff
@@ -1,9 +1,26 @@
 import * as React from 'react';
 import './LoginPage.css';
+import InputComponent from '../../components/InputComponent';
+import Footer from '../../components/footer';
+import Button from '../../components/Button';
+import { Header } from '../../components';
 const LoginPage = () => {
```
Comment (Chetan-MckinseyBEBR) [inline src/pages/LoginPage/index.jsx:7]: new line

**Review by Chetan-MckinseyBEBR — COMMENTED** [review]
Added few comments

**Pull request merged**